webapp/www/templates/home.html

```diff
@@ -45,43 +45,43 @@
 <fullpage class="instant-size" page-class="page">
```

Comment:
I removed the fullpage and set the layout to cards but a bug that I am trying to fix is linking nav bar to sections of the homepage.